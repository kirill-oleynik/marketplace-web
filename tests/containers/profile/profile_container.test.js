const React = require('react');
const renderer = require('react-test-renderer');
const { ProfileContainer } = require('../../../app/containers/profile/profile_container');

jest.mock('../../../app/components/profile/profile_modal', () => 'ProfileModal');

describe('#render', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(
      <ProfileContainer
        currentUser={{}}
        profileModalActive={false}
        toggleProfileModal={() => {}}
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
