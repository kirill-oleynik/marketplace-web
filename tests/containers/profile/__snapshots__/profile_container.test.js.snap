// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<ProfileModal
  closeModal={[Function]}
  currentUser={Object {}}
  isOpen={false}
/>
`;
